ansible/tasks/oauth2-proxy.yml

```diff
@@ -0,0 +1,29 @@
+- name: "download oauth2_proxy"
+  get_url:
+    src: "https://github.com/bitly/oauth2_proxy/releases/download/v2.2/oauth2_proxy-2.2.0.linux-amd64.go1.8.1.tar.gz"
```

Comment:
Do we want to hardcode the version here? I don't think we do that in any task currently.

Reply:
Yes I'm lazy

Reply:
Especially considering every time you bump the version, you need to update the hash anyway.